src/components/EditRegistration.tsx

```diff
@@ -43,7 +45,16 @@ const EditRegistration = ({
           </p>
         </>
       ) : (
-        <p className="EditRegistration__title">Edit</p>
+        <div className="EditRegistration__header">
+          <p className="EditRegistration__title">Edit</p>
+          <Button
+            type="link"
+            className="EditRegistration__closeModalButton"
+            onClick={() => setRegistrationPopoverVisible(false)}
+          >
+            Cancel
+          </Button>
+        </div>
```

Comment:
I know we didn't have a visual design for this, but I think we should put it down in the bottom right corner instead of the top right.

That combined with setting the icon to toggle should help get people in and out of the modal.

Thoughts?

![image](https://user-images.githubusercontent.com/1252199/155549660-d35060e9-5d52-43c3-8892-fba5f46ab8b4.png)